sae_lens/training/activations_store.py

```diff
@@ -49,13 +49,77 @@ class ActivationsStore:
     _storage_buffer: torch.Tensor | None = None
     device: torch.device
 
+    @classmethod
+    def _from_save_activations(
+        cls,
+        model: HookedRootModule,
+        cfg: CacheActivationsRunnerConfig,
+    ) -> "ActivationsStore":
```

Comment:
Why is this different than `from_cache_activations` below? These both take a cache activations runner config and return a store